packager/packager.js

```diff
@@ -198,11 +203,15 @@ function statusPageMiddleware(req, res, next) {
 }
 
 function getAppMiddleware(options) {
+  var transformerPath = options.transformer;
+  if (!path.isAbsolute(transformerPath)) {
```

Comment:
We want to support v0.10 (after all jest doesn't run on v0.12 yet). We have a package installed for this `absolute-path`